doc/classes/ProjectSettings.xml

```diff
@@ -584,14 +586,6 @@
 		<member name="memory/limits/multithreaded_server/rid_pool_prealloc" type="int" setter="" getter="">
 			This is used by servers when used in multi threading mode (servers and visual). RIDs are preallocated to avoid stalling the server requesting them on threads. If servers get stalled too often when loading resources in a thread, increase this number.
 		</member>
-		<member name="mono/debugger_agent/port" type="int" setter="" getter="">
```

Comment:
Mono options should be kept. (Also in `@GlobalScope`). They're sadly removed if running doctool with a non-Mono build, but it's easy enough to undo the removal.